composer.json

```diff
@@ -46,5 +46,8 @@
     },
     "require-dev": {
         "phpunit/phpunit": ">=3.3.0"
```

Comment:
I'm always slightly suspicious of these things apparently working with such old versions... Considering how much these packages like to change things

Reply:
They are, only the output of the test-runner changes a bit:

* PHPUnit 4.8.36 OK (113 tests, 270 assertions)
* PHPUnit 12.0.2 OK, but there were issues! / Tests: 113, Assertions: 270, PHPUnit Deprecations: 1.

See here: https://github.com/ktomk/Net_URL2/actions/runs/13295699011